sycl/test-e2e/DeviceIsIntegratedGPU/device_is_integrated_gpu.cpp

```diff
@@ -0,0 +1,29 @@
+//==- device_is_integrated_gpu.cpp - sycl_ext_oneapi_device_is_integrated_gpu
```

Comment:
formatting looks broken

Reply:
It looks like there is no way to make it in one line with the same amount of information, so it was splitted into two.

Reply:
it is. maybe it makes sense to remove this info. have no idea why we need to duplicate file name, for example. Or why extension name can't be just mentioned in the comment describing the purpose of test.